app/src/main/java/currencyConverter/model/CurrencyCode.java

```diff
@@ -0,0 +1,96 @@
+package currencyConverter.model;
+
+import java.util.Arrays;
+import java.util.List;
+
+public enum CurrencyCode {
```

Comment:
I think we shouldn't hardcode currency's info in any form. It should be an external file, that is being updated with info from some qualified server.
Let it be for now, for alpha demonstrative version.